evals/cli/oaievalset.py

```diff
@@ -61,17 +61,31 @@ def get_parser() -> argparse.ArgumentParser:
     return parser
 
 
+class OaiEvalSetArguments(argparse.Namespace):
+    model: str
+    eval_set: str
+    resume: bool
+    exit_on_error: bool
+
+
 def run(
-    args, unknown_args, registry: Optional[Registry] = None, run_command: str = "oaieval"
+    args: OaiEvalSetArguments,
+    unknown_args: list[str],
+    registry: Optional[Registry] = None,
+    run_command: str = "oaieval",
 ) -> None:
     registry = registry or Registry()
     commands: list[Task] = []
-    eval_set = registry.get_eval_set(args.eval_set)
-    for eval in registry.get_evals(eval_set.evals):
-        command = [run_command, args.model, eval.key] + unknown_args
-        if command in commands:
-            continue
-        commands.append(command)
+    eval_set = registry.get_eval_set(args.eval_set) if args.eval_set else None
```

Comment:
We should log an error when the eval set is not found, to make it more clear to the user what happened.

Reply:
Done, https://github.com/openai/evals/pull/1027/files#diff-1e7ed1aa6747e1064fcaa1ee71be6b30ebbe4d718776e1ffb4cc577cd44d8ab6R85 and https://github.com/openai/evals/pull/1027/files#diff-1e7ed1aa6747e1064fcaa1ee71be6b30ebbe4d718776e1ffb4cc577cd44d8ab6R92